### PR TITLE
Fixes #28 Type annotations

### LINKED ISSUE #28: Add type annotations
The project is Py3.6+ which means we can add type annotations everywhere. Mypy is already installed, so let's do this.

### PR DESCRIPTION
**Pull request by ewjoachim**
Fixes #28

Also changes the rule that says "no multiple imports on the same line

### Successful PR Checklist:
- [ ] Tests < nothing to test
- [ ] Documentation (optionally: [run spell checking](https://github.com/peopledoc/septentrion/blob/master/CONTRIBUTING.rst#build-the-documentation)) < The doc doesn't exist yet
- [ ] Had a good time contributing? (feel free to give some feedback) < I have to admit it was subpar...

While going through all the code, I'm definitely convinced a few refactors would simplify a lot of things. The modelisation was subpar, and I say this as one of the main contributors of this lib. Amazing how one or two years gives you perspectives on coding habits...

### CODE REVIEW

File: septentrion/runner.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def close(self) -> None:
             raise SQLRunnerException("Block closed !")
         self.closed = True
 
-    def run(self, cursor: psycopg2.extentions.cursor) -> int:
+    def run(self, cursor: cursor) -> int:
```
Comment (mgu) [inline septentrion/runner.py]: the cursor parameter overrides the cursor object imported. There is no problem here but is it a good practice ?

Comment (ewjoachim) [reply septentrion/runner.py]: oh gosh you're right ! Not sure what the rest of the world thinks of it but I hate it, it creates ambiguity.

Comment (k4nar) [reply septentrion/runner.py]: maybe we could do `from psycopg2.extensions.cursor import cursor as Cursor` ?

Comment (ewjoachim) [reply septentrion/runner.py]: yes, that was my planned fix. This or `as cursor_class`, but what you suggested was my number 1.

**Comment by ewjoachim** [comment]
@mgu ?

**Pull request merged**